src/pause/pause_map.c

```diff
@@ -21,94 +21,24 @@ void pause_map_handle_input(MenuPanel* tab);
-    { .pos = { .x =  66, .y = 289 }, .parent =  0, .pathLength = 21, .path = gPauseMapPaths[19], .afterRequirement = STORY_CH5_ZIP_LINE_READY,          .id = LOCATION_JADE_JUNGLE },
-    { .pos = { .x =  84, .y = 274 }, .parent = 19, .pathLength =  2, .path = gPauseMapPaths[20], .afterRequirement = STORY_61,                          .id = LOCATION_YOSHIS_VILLAGE },
-    { .pos = { .x =  47, .y = 252 }, .parent = 19, .pathLength =  5, .path = gPauseMapPaths[21], .afterRequirement = STORY_CH5_BEGAN_PEACH_MISSION,     .id = LOCATION_MT_LAVALAVA },
-    { .pos = { .x =  59, .y = 126 }, .parent =  0, .pathLength =  0, .path = gPauseMapPaths[22], .afterRequirement = STORY_CH6_BEGAN_PEACH_MISSION,     .id = LOCATION_FLOWER_FIELDS },
-    { .pos = { .x =  58, .y =  92 }, .parent = 22, .pathLength =  4, .path = gPauseMapPaths[23], .afterRequirement = STORY_CH6_BEGAN_PEACH_MISSION,     .id = LOCATION_CLOUDY_CLIMB },
-    { .pos = { .x = 183, .y = 125 }, .parent =  3, .pathLength =  0, .path = gPauseMapPaths[24], .afterRequirement = STORY_CH7_MAYOR_MURDER_SOLVED,     .id = LOCATION_SHIVER_CITY },
-    { .pos = { .x = 203, .y = 126 }, .parent = 24, .pathLength =  1, .path = gPauseMapPaths[25], .afterRequirement = STORY_CH7_DEFEATED_MONSTAR,        .id = LOCATION_SHIVER_SNOWFIELD },
-    { .pos = { .x = 234, .y = 128 }, .parent = 25, .pathLength =  2, .path = gPauseMapPaths[26], .afterRequirement = STORY_CH7_GOT_SNOWMAN_SCARF,       .id = LOCATION_STARBORN_VALLEY },
-    { .pos = { .x = 220, .y = 112 }, .parent = 25, .pathLength =  2, .path = gPauseMapPaths[27], .afterRequirement = STORY_CH7_STAR_SPRIT_DEPARTED,     .id = LOCATION_SHIVER_MOUNTAIN },
-    { .pos = { .x = 223, .y =  99 }, .parent = 27, .pathLength =  1, .path = gPauseMapPaths[28], .afterRequirement = STORY_CH7_STAR_SPRIT_DEPARTED,     .id = LOCATION_CRYSTAL_PALACE },
-    { .pos = { .x = 158, .y = 110 }, .parent =  2, .pathLength =  6, .path = gPauseMapPaths[29], .afterRequirement = STORY_61,                          .id = LOCATION_STAR_WAY },
-    { .pos = { .x = 163, .y =  78 }, .parent = 29, .pathLength =  6, .path = gPauseMapPaths[30], .afterRequirement = STORY_61,                          .id = LOCATION_STAR_HAVEN },
-    { .pos = { .x =  96, .y =  49 }, .parent = 30, .pathLength = 13, .path = gPauseMapPaths[31], .afterRequirement = STORY_EPILOGUE,                    .id = LOCATION_BOWSERS_CASTLE },
-    { .pos = { .x =  88, .y =  22 }, .parent = 31, .pathLength =  3, .path = gPauseMapPaths[32], .afterRequirement = STORY_EPILOGUE,                    .id = LOCATION_PEACHS_CASTLE },
-    { .pos = { .x =  98, .y = 147 }, .parent =  0, .pathLength =  0, .path = gPauseMapPaths[33], .afterRequirement = STORY_61,                          .id = LOCATION_MARIOS_HOUSE }
-};
 
-s32 gPauseMapArrowWobble[] = { 0, 2, 3, 3, 4, 4, 4, 4, 3, 2, 1, 0 };
-MenuWindowBP gPauseMapWindowBPs[] = {
+#include "gen/world_map.h"
```

Comment:
Header files shouldn't contain declarations, so this would be an .inc.c rather than a .h imo